cmd/srcd/cmd/table.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+)
+
+func PrintTable(tw *tabwriter.Writer, header []string, formats []string, data [][]interface{}) error {
+	if len(header) != len(formats) {
+		return fmt.Errorf("header and formats have different sizes")
+	}
+
+	sHeader := strings.Join(header, "\t") + "\n"
+	fmt.Fprintf(tw, sHeader)
+
+	sFormats := strings.Join(formats, "\t") + "\n"
+	for _, row := range data {
+		fmt.Fprintf(tw, sFormats, row...)
+	}
+
+	return tw.Flush()
+}
+
+func PrintDefaultTable(output io.Writer, header []string, formats []string, data [][]interface{}) error {
```

Comment:
what is the use case for `PrintTable`?

Reply:
Currently none 😅. So yes, I guess that we can simply rename `PrintDefaultTable` into `PrintTable`  and remove the current one.